exercise/function/calculation.c

```diff
@@ -0,0 +1,52 @@
+/* Write sum, sub, multiple, divide calculation */
+
+#include <stdio.h>
+
+/* Function Prototype */
+
+int mul(int a, int b);
+int sub(int a, int b);
+int add(int a, int b);
+int div(int a, int b);
+
+int main(void)
+{
+    int a, b;
```

Comment:
- [ ] Follow this rule for each function: Separate declaration/definition and the first statement by an empty line.